src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -65,14 +65,17 @@ parseTypeSynonymDeclaration =
 
 parseValueDeclaration :: TokenParser Declaration
 parseValueDeclaration = do
-  name <- parseIdent
-  binders <- P.many parseBinderNoParens
-  value <- Left <$> (indented *>
-                       P.many1 ((,) <$> parseGuard
-                                    <*> (indented *> equals *> parseValueWithWhereClause)
-                               ))
-       <|> Right <$> (indented *> equals *> parseValueWithWhereClause)
-  return $ ValueDeclaration name Public binders value
+  binders <- (:) <$> parseBinderAtom <*> (P.many parseBinderNoParens)
```

Comment:
What's the difference between `parseBinder` and `parseBinderAtom`? I have just tried with `parseBinder` and it breaks many modules from being parsed.